src/ansys/fluent/core/solver/flobject.py

```diff
@@ -619,7 +617,7 @@ class Map(SettingsBase[DictStateType]):
 
 
 def _get_new_keywords(obj, kwds):
-    newkwds = {}
+    newkwds = dict()
```

Comment:
So... I know I suggested this change but I think it can be reverted.   See the PyFluent dev channel for a link I shared.